circle.yml

```diff
@@ -9,6 +9,11 @@ checkout:
   post:
     # Update the `master` branch. Otherwise CircleCI will give us a cached one.
     - git fetch -u origin +master:master
+    # Checkout the merged PR for testing as CircleCI will just use the PR head otherwise.
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/head:pr/$CIRCLE_PR_NUMBER/head ; fi
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git fetch origin +refs/pull/$CIRCLE_PR_NUMBER/merge:pr/$CIRCLE_PR_NUMBER/merge ; fi
+    - if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then git checkout -qf pr/$CIRCLE_PR_NUMBER/merge ; fi
```

Comment:
Checks out the merge of the PR with the base branch. Could be useful to get infrastructure fixes. Though it obviously won't include fixes to `circle.yml` as it has already been parsed.

xref: https://discuss.circleci.com/t/reparse-circle-yml-after-checkout-post/6187